TramsDataApi/Gateways/TrustGateway.cs

```diff
@@ -29,10 +29,12 @@ public IList<Group> SearchGroups(string groupName, string ukprn, string companie
             {
                 return _dbContext.Group.OrderBy(group => group.GroupUid).Skip((page - 1) * 10).Take(10).ToList();
             }
-            
+
+            var groupToCheck = groupName?.ToLower();
+
             return _dbContext.Group
                 .Where(g => (
-                    g.GroupName == groupName ||
+                    g.GroupName.ToLower().Contains(groupToCheck) ||
```

Comment:
I'm pretty sure `.Contains` is mapped to `LIKE` so is not case sensitive. 
Are you sure you need a ToLower here?